README.md

```diff
@@ -35,10 +35,12 @@ export EDITOR=vim # flamewar starting in 3, 2, 1...
 Now, execute the following commands:
 ```sh
 export DJANGO_SETTINGS_MODULE=gutenberg.settings.${GUTENBERG_ENV}_settings
+git clone https://github.com/KSIUJ/gutenberg.git
```

Comment:
This was valid — you usually want your virtualenv to live outside your project root directory (so that e.g. removing and re-cloning the repository does not remove your environment).